core/chains/evm/logpoller/log_poller.go

```diff
@@ -435,3 +435,11 @@ func (lp *LogPoller) LatestLogByEventSigWithConfs(eventSig common.Hash, address
 	}
 	return log, nil
 }
+
+func (lp *LogPoller) LatestLogEventSigsAddrs(fromBlock int64, eventSigs []common.Hash, addresses []common.Address, qopts ...pg.QOpt) ([]Log, error) {
+	logs, err := lp.orm.LatestLogEventSigsAddrs(fromBlock, addresses, eventSigs, qopts...)
+	if err != nil {
+		return nil, err
+	}
+	return logs, nil
```

Comment:
Could return directly?
```suggestion
	return lp.orm.LatestLogEventSigsAddrs(fromBlock, addresses, eventSigs, qopts...)
```

Reply:
Done